src/modules/ComicList/container/ComicsContainer.tsx

```diff
@@ -0,0 +1,53 @@
+import React, {useEffect, useState} from 'react';
+import { List } from '@mui/material';
+import IComic from '../model/Comic';
+import Comic from '../component/Comic'
+import useAPI from '../../common/hooks/UseApi';
+
+interface ComicPage {
+    offset: number,
+    limit: number,
+    total: number,
+    count: number,
+    results: IComic[]
+}
+
+interface RequestParameters {
+    [key: string]: string;
+}
+
+function ComicsContainer() {
+    const [options, setOptions] = useState<RequestParameters>({orderBy: 'title', limit: '20', offset: '0'})
```

Comment:
Do you need a generic type here if you already know the parameters? 
```suggestion
    const [options, setOptions] = useState({orderBy: 'title', limit: '20', offset: '0'})
```